src/intrinsics.cpp

```diff
@@ -128,7 +128,7 @@ static Type *INTT(Type *t)
         return T_int16;
     unsigned nb = t->getPrimitiveSizeInBits();
     assert(t != T_void && nb > 0);
-    return IntegerType::get(jl_LLVMContext, nb);
+    return IntegerType::get(t->getContext(), nb);
```

Comment:
Cleanup comment for after, hoist the `ctx` load to the beginning of the function since it is going to be used in all the branches